styles/styles.css

```diff
@@ -23,11 +23,11 @@
 
 img {
     max-width: 200px;
-    padding-right: 1rem;
+    /* padding-right: 1rem; */
 }
 
 button {
-    margin-right: 2rem;
+    /* margin-right: 2rem; */
```

Comment:
You can delete these instead of commenting them out, unless you want to use them later.